modelgauge/records.py

```diff
@@ -29,7 +29,7 @@ class TestRecord(BaseModel):
     test_initialization: InitializationRecord
     dependency_versions: Mapping[str, str]
     sut_uid: str
-    sut_initialization: InitializationRecord
+    sut_initialization: Optional[InitializationRecord]
```

Comment:
`sut_initialization` shouldn't be changed to be optional. I think you should just define a custom object you would like to be returned by the annotator runner.